webapp/packages/core-utils/src/copyToClipboard.test.ts

```diff
@@ -0,0 +1,26 @@
+import { copyToClipboard } from './copyToClipboard';
+
+document.execCommand = jest.fn();
+
+describe('copyToClipboard', () => {
+  it('should copy data to clipboard', () => {
+    copyToClipboard('test');
+
+    expect(document.execCommand).toHaveBeenCalledWith('copy');
+  });
+
+  it('should focus on active element after copy', () => {
+    document.body.focus = jest.fn();
```

Comment:
and this

Reply:
this gonna be fine. jest does that automatically